array-methods.js

```diff
@@ -49,10 +55,17 @@ var datasetWithRoundedDollar = null;
     }
   assign the resulting new array to `roundedDime`
 */
-var datasetWithRoundedDime = null;
+
+var datasetWithRoundedDime = dataset.bankBalances.map( ( acc ) => {
+    return { "amount" : acc.amount, "state" : acc.state, "roundedDime" : parseFloat(Number(acc.amount).toFixed(1)) };
```

Comment:
Using toFixed is probably not the best idea. Given the number 12.35 this will return 12.3 when Math.round will return 12.4